shared/src/dto/trainee/trainee.dto.ts

```diff
@@ -55,12 +94,19 @@ export class DtoUpdateTrainee {
 
   @ApiProperty()
   @IsOptional()
-  pesel?: string;
+  @IsNumberString({ no_symbols: true })
+  @Length(11, 11)
+  pesel?: string | null;
 
   @ApiProperty()
   @IsOptional()
   pkk?: string;
 
+  @ApiProperty()
+  @IsNotEmpty()
+  @Type(() => Date)
```

Comment:
Proper validation for this being a date is missing. Please take a look at how I did it in the `availability.dto.ts` file.
Also take a look at how I have used the `@ApiProperty()` decorator